simpegEM/FDEM/FDEM.py

```diff
@@ -8,6 +8,102 @@ def omega(freq):
     """Change frequency to angular frequency, omega"""
     return 2.*np.pi*freq
 
+def getSource(self,freq):
```

Comment:
Is this really getRHS? Should this be implemented in the transmitter?

Also talking to brendan, and looking at flow problems, and MT, the Tx name is slightly confusing. It might be better to call it source?

```
for source in sources:
    rhs = source.getRHS(mesh)
```

Thoughts on changing this?
-R